src/04-modality-specific-files/02-magnetoencephalography.md

```diff
@@ -287,47 +287,47 @@ EEG, head localization coils, and anatomical landmarks.
+| DigitizedHeadPointsCoordinateUnits             | OPTIONAL              | [string][]    | Units of the coordinates of `DigitizedHeadPointsCoordinateSystem`. MUST be `m`, `cm`, or `mm`.                                                                                                                                                                                   |
+| DigitizedHeadPointsCoordinateSystemDescription | OPTIONAL              | [string][]    | Freeform text description or link to document describing the Digitized head Points coordinate system system in detail.                                                                                                                                                           |
 
 Anatomical MRI:
 
-| Field name  | Description                                                                                                                                                                                                                                                                                          |
-| ---------------------------------------------------------------------------------------------------------------------------------- | --------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
-| IntendedFor | OPTIONAL. Path or list of path relative to the subject subfolder pointing to the structural MRI, possibly of different types if a list is specified, to be used with the MEG recording. The path(s) need(s) to use forward slashes instead of backward slashes (e.g. `ses-/anat/sub-01_T1w.nii.gz`). |
+| **Key name** | **Requirement level** | **Data type**                          | **Description**                                                                                                                                                                                                                                                                             |
+| ------------ | --------------------- | -------------------------------------- | ------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------------- |
+| IntendedFor  | OPTIONAL              | [string][] or [array][] of [strings][] | Path or list of path relative to the subject subfolder pointing to the structural MRI, possibly of different types if a list is specified, to be used with the MEG recording. The path(s) need(s) to use forward slashes instead of backward slashes (e.g., `ses-/anat/sub-01_T1w.nii.gz`). |
```

Comment:
It looks like the `strings` link isn't defined in this page.

Reply:
ahh good catch, thanks! Fixed in https://github.com/bids-standard/bids-specification/pull/605/commits/e87497cb9fe7691d61518b1c0efcc193f396fc40